core/src/main/java/org/fao/geonet/MetadataResourceDatabaseMigration.java

```diff
@@ -66,20 +66,24 @@
 public class MetadataResourceDatabaseMigration implements DatabaseMigrationTask {
 
     private static final ArrayList<Namespace> NAMESPACES =
-        Lists.newArrayList(
-            ISO19139Namespaces.GMD,
```

Comment:
These three lines shouldn't be in the commit as only white spaces are modified.

Even if the indentation is wrong, we try not to modify it unless we modify the content of the line.

Same with other lines on this commit.

Reply:
Should be fine now. Please go ahead.